library/network/src/lib/network/firewalld.rb

```diff
@@ -0,0 +1,324 @@
+# encoding: utf-8
+#
+# ***************************************************************************
+#
+# Copyright (c) 2016 SUSE LLC.
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of version 2 or 3 of the GNU General
+# Public License as published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.   See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE about this file by physical or electronic mail,
+# you may find current contact information at www.suse.com
+#
+# ***************************************************************************
+#
+# File: lib/network/firewalld.rb
+# Summary:  FirewallD configuration API
+# Authors:  Karol Mroz <kmroz@suse.de>, Markos Chandras <mchandras@suse.de>
+#
+
+require 'yast'
+
+module Firewalld
+
+  # exception when firewall-cmd fails
+  class FirewallCMDError < StandardError
+  end
+
+  class FWCmd
+
+    include Yast::Logger
+
+    BASH_SCR_PATH = Yast::Path.new(".target.bash_output")
+    # Base firewall-cmd command
+    COMMAND = "LANG=Python firewall-cmd"
```

Comment:
`LANG=Python`? Is this to avoid parsing translated texts? `LANG=C` would be better.